### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by yoptar**
- Configuration and model for Chinese SQUAD based on Chinese BERT
- Remove all Keras usage in favor of `tensorflow.keras`
- Start refactoring of goal-oriented bot code
- Update dependencies versions
- Add a BERT-embedding component as a first step of moving from [google-research/bert](https://github.com/google-research/bert) to [HuggingFace's Transformers](https://github.com/huggingface/transformers)
- Smaller changes and fixes

### CODE REVIEW

File: docs/features/models/squad.rst [hunk 1]
```diff
@@ -218,3 +218,26 @@ Link to SDSJ Task B dataset: http://files.deeppavlov.ai/datasets/sber_squad-v1.1
...
+DRCD
+~~~~~~~~~~~
+
+Pretrained models are available and can be downloaded:
+
+.. code:: bash
+
+    python -m deeppavlov download deeppavlov/configs/squad/squad_zh_bert.json
+	
+	python -m deeppavlov download deeppavlov/configs/squad/squad_zh_zh_bert.json
```
Comment (IgnatovFedor) [inline docs/features/models/squad.rst]:
```suggestion
    python -m deeppavlov download deeppavlov/configs/squad/squad_zh_zh_bert.json
```

File: docs/features/models/squad.rst [hunk 2]
```diff
@@ -218,3 +218,26 @@ Link to SDSJ Task B dataset: http://files.deeppavlov.ai/datasets/sber_squad-v1.1
...
+
+DRCD
+~~~~~~~~~~~
+
+Pretrained models are available and can be downloaded:
+
+.. code:: bash
+
+    python -m deeppavlov download deeppavlov/configs/squad/squad_zh_bert.json
+
```
Comment (IgnatovFedor) [inline docs/features/models/squad.rst]:
```suggestion

```

File: docs/features/models/squad.rst (hunk 1 above)
Comment (IgnatovFedor) [reply docs/features/models/squad.rst]: 4 spaces instead of 1 tab

File: docs/features/models/syntaxparser.rst [hunk 3]
```diff
@@ -167,4 +167,4 @@ and dependency head.
 .. _`UD Pipe Future`: https://github.com/CoNLL-UD-2018/UDPipe-Future
 .. _`UDify (multilingual BERT)`: https://github.com/hyperparticle/udify
 
-So our model is by a valuable margin the state-of-the-art system for Russian syntactic parsing.
\ No newline at end of file
+So our model is by a valuable margin the state-of-the-art system for Russian syntactic parsing.
```
Comment (IgnatovFedor) [inline docs/features/models/syntaxparser.rst]:
```suggestion
So our model is the state-of-the-art system for Russian syntactic parsing by a valuable margin.
```
I do not insist, but the original sentence hurts the eye

File: deeppavlov/models/classifiers/keras_classification_model.py [hunk 4]
```diff
@@ -284,8 +256,8 @@ def _load(self, model_name: str) -> None:
                 try:
                     model.load_weights(str(weights_path))
                 except ValueError:
-                    raise ConfigError(
-                        "Some non-changable parameters of neural network differ from given pre-trained model")
+                    raise ConfigError("Some non-changeable parameters of neural network differ"
```
[the remark this answers is not in the document]
Comment (yoptar) [reply deeppavlov/models/classifiers/keras_classification_model.py]: I think this would change the intended meaning.

**Pull request merged**